lib/relay/relay.go

```diff
@@ -18,15 +18,26 @@ import (
 	"github.com/syncthing/relaysrv/protocol"
 	"github.com/syncthing/syncthing/lib/config"
 	"github.com/syncthing/syncthing/lib/discover"
-	"github.com/syncthing/syncthing/lib/model"
 	"github.com/syncthing/syncthing/lib/osutil"
 	"github.com/syncthing/syncthing/lib/sync"
 
 	"github.com/thejerf/suture"
 )
 
+type Svc struct {
```

Comment:
This is just moved to above the New as is common, plus changed the type of the conns channel.